alpha/action/render_test.go

```diff
@@ -792,7 +792,7 @@ func generateSqliteFile(path string, imageMap map[image.Reference]string) error
 		return err
 	}
 
-	populator := registry.NewDirectoryPopulator(loader, graphLoader, dbQuerier, imageMap, nil, false)
+	populator := registry.NewDirectoryPopulator(loader, graphLoader, dbQuerier, imageMap, nil)
```

Comment:
Completely unrelated, but this likely signals that we should be accepting some sort of configuration struct if we're accepting this number of argument, namely for readability as it's not immediately clear aside from the variable names what the function signature expects.